ethcore/src/miner/miner.rs

```diff
@@ -478,35 +491,32 @@ impl MinerService for Miner {
 		self.gas_range_target.read().unwrap().1
 	}
 
-	fn import_transactions<T>(&self, chain: &MiningBlockChainClient, transactions: Vec<SignedTransaction>, fetch_account: T) ->
-		Vec<Result<TransactionImportResult, Error>>
-		where T: Fn(&Address) -> AccountDetails {
-		let results: Vec<Result<TransactionImportResult, Error>> = {
-			let mut transaction_queue = self.transaction_queue.lock().unwrap();
-			transactions.into_iter()
-				.map(|tx| transaction_queue.add(tx, &fetch_account, TransactionOrigin::External))
-				.collect()
-		};
-		if !results.is_empty() && self.options.reseal_on_external_tx && 	self.tx_reseal_allowed() {
+	fn import_external_transactions(&self, chain: &MiningBlockChainClient, transactions: Vec<SignedTransaction>) ->
+		Vec<Result<TransactionImportResult, Error>> {
+
+		let mut transaction_queue = self.transaction_queue.lock().unwrap();
+		let results = self.add_transactions_to_queue(chain, transactions, TransactionOrigin::External,
```

Comment:
We use a bit different indentation style. I think I would write something like this here:
```rust
let results = self.add_transactions_to_queue(
    chain, transactions, TransactionOrigin::External, &mut transaction_queue
);
```

Or would just keep it single line.

(Same in L693)